dhclient-ipv6

```diff
@@ -49,6 +49,7 @@ EXT_IFACE='eth0'
 INT_IFACE='eth1'
 OTHER_IFACES=''
 RADVD_TEMPLATE='/etc/radvd.conf.tmpl'
+DNSMASQ_TEMPLATE='/etc/dnsmasq.template'
```

Comment:
sorry I missed this the first time around. Can we be consistent here and call this `/etc/dnsmasq.conf.tmpl`?